actor-profile.js

```diff
@@ -64,11 +83,13 @@ class ActorProfile extends HTMLElement {
     }
 
     if (actorInfo.preferredUsername) {
-      const pUserName = document.createElement('a')
+      const pUserName = document.createElement('div')
       pUserName.classList.add('profile-username')
-      pUserName.href = db.getObjectPage(actorInfo)
```

Comment:
Why was this removed? Could we add it back? I find it useful to be able to navigate to accounts on their native web url.

Reply:
Added back.
It was a mistake.